examples/flink-consumer-source/src/main/scala/org/apache/flink/batch/connectors/pulsar/example/README.md

```diff
@@ -279,5 +279,67 @@ Please find sample output for above application as follows:
 
 ### Complete Example
 
-You can find a complete example [here](https://github.com/apache/incubator-pulsar/tree/master/pulsar-flink/src/test/scala/org/apache/flink/batch/connectors/pulsar/example/FlinkPulsarBatchJsonSinkScalaExample.scala).
+You can find a complete example [here](https://github.com/apache/pulsar/tree/master/examples/flink-consumer-source/src/main/scala/org/apache/flink/batch/connectors/pulsar/example/FlinkPulsarBatchJsonSinkScalaExample.scala).
 In this example, Flink DataSet is processed and written to Pulsar in Json format.
+
+
+# PulsarJsonOutputFormat
```

Comment:
@ambition119 Firstly, thanks for the contribution.
Following documentation should point `PulsarAvroOutputFormat` documentation.
Also row: **289** and **304**.

Reply:
ok